lua/terrortown/lang/de.lua

```diff
@@ -1595,307 +1594,305 @@ L.label_roles_credits_award_repeat = "Mehrfache Belohnung"
 L.label_roles_newroles_enabled = "Aktiviere eigene Rollen"
 L.label_roles_max_roles = "Obere Grenze für Rollen"
 L.label_roles_max_roles_pct = "Prozentuale obere Grenze für Rollen"
---L.label_roles_max_baseroles = "Upper base role limit"
---L.label_roles_max_baseroles_pct = "Upper base role limit by percentage"
---L.label_detective_hats = "Enable hats for policing roles like the Detective (if player model allows to have them)"
+L.label_roles_max_baseroles = "Obere Basisrollenbegrenzung"
+L.label_roles_max_baseroles_pct = "Obere Basisrollenbegrenzung in Prozent"
+L.label_detective_hats = "Aktiviere Hüte für Polizeirollen wie den Detektiv (sofern das Spielermodell dies zulässt)"
```

Comment:
```suggestion
L.label_detective_hats = "Aktiviere Hüte für bekannte Kontrollollen wie den Detektiv (sofern das Spielermodell dies zulässt)"
```

Ich habe diese Rollen in meinem Bodysearch Rework "bekannte Kontrollrollen" genannt. Können auch deine Lösung nehmen, aber wir sollten das besprechen

Reply:
nachdem ich deine komplette Übersetzung gelesen habe, fände ich vermutlich `bekannte Polizeirollen` am besten hierfür